perl/p5-test-cpan-meta-json/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26 5.28 5.30
+perl5.setup         Test-CPAN-Meta-JSON 0.16 ../../authors/id/B/BA/BARBIE
+revision            0
+checksums           rmd160  2c188894b62e906b491e0ea7a67ee0f47ce56a40 \
+                    sha256 67ac509adffb1d2b256a8f8c0523e00761d960166192c6070298f7088a9ae9c9 \
+                    size 21892
+
+platforms           darwin
+
+license             Artistic-2
+maintainers         {isi.edu:calvin @cardi} openmaintainer
+description         Test::CPAN::Meta::JSON - Validate a META.json file within a CPAN distribution.
+long_description    ${description}
+
+if {${perl5.major} != ""} {
```

Comment:
Nitpick: `ne` is preferred over `!=` for non-numeric exact string comparison. (I proposed changing this in cpan2port's output: macports/macports-contrib#12)

Reply:
Thanks! I updated the Portfile with your suggestion.